_includes/templates/iot-gateway/request-converter-json-config.md

```diff
@@ -26,37 +32,49 @@ Mapping subsection will look like:
 
 ```json
     {
-      "url": "getdata",
-      "httpMethod": "GET",
-      "httpHeaders": {
-        "ACCEPT": "application/json"
+  "url": "getdata",
```

Comment:
Please return spaces, to demonstrate the real position of this section in the configuration.